Core/Object Arts/Dolphin/MVP/Views/Common Controls/UI.ListView.cls

```diff
@@ -1318,7 +1318,7 @@ lvmInsertItem: aLvItem
 		lpParam: aLvItem asParameter!
 
 lvmRedrawItems: firstIndex to: lastIndex
-	"Private - Insert aLvItem into the tree view and add it to our contents dictionary"
+	"Private - Cause redraw messages to be sent for each of these item entries"
```

Comment:
This is about right, but there is only really any value in comments in these methods that are just sending the windows message if it adds something over and above the Microsoft documentation. In this case it doesn't, so I would just delete it.

Reply:
Right, I knew it was either this comment or deletion. Deletion it'll be.